kitsune/questions/utils.py

```diff
@@ -45,12 +44,4 @@ def get_mobile_product_from_ua(user_agent):
         return "ios"
 
     # android
-    try:
-        # We are using firefox instead of Firefox as lower() has been applied to the UA
-        mobile_client = re.search(r"firefox/(?P<version>\d+)\.\d+", ua).groupdict()
-    except AttributeError:
-        return None
-    else:
-        if int(mobile_client["version"]) >= 69:
-            return "firefox-preview"
```

Comment:
I think it's only this block we want to remove, we need try/except above so non-firefox browsers on android return `None` - I think this is why the tests are failing.